### PR TITLE
nuclear fission madness

### PR DESCRIPTION
**Pull request by Rundas01**
will be a draft at first. that will be changed once it's finished.

### CODE REVIEW

**Comment by Trinsdar** [comment]
Fyi gui textures are going to have to be changed based on the old textures i put in instead, don't worry about the modern textures it's been using, i'll fix those in the included resource pack

File: common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java [hunk 1]
```diff
@@ -233,4 +232,17 @@ private static void plasticPolymerization(Material monomer, Material polymer, in
...
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Uranium,1))).fi(LeachingSolution.getLiquid(1000)).fo(LeachedUranium.getLiquid(1000)).add("uranium_leaching",120,1000);
+
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(TrithoriumOctoxide,11))).fi(NitricAcid.getLiquid(3000)).fo(Thoriumdioxidedinitrate.getLiquid(3000),AntimatterMaterials.Water.getLiquid(4000),NitrogenDioxide.getGas(2000)).add("thoriumdioxidedinatrate",120,1000);
+        CHEMICAL_REACTING.RB().fi(Thoriumdioxidedinitrate.getLiquid(10000),Ammonia.getGas(22000)).fo(Ammoniumdithoranate.getLiquid(5000)).add("uranium_leaching",120,1000);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Uranium,1))).fi(LeachingSolution.getLiquid(1000)).fo(LeachedUranium.getLiquid(1000)).add("uranium_leaching",120,1000);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Uranium,1))).fi(LeachingSolution.getLiquid(1000)).fo(LeachedUranium.getLiquid(1000)).add("uranium_leaching",120,1000);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Uranium,1))).fi(LeachingSolution.getLiquid(1000)).fo(LeachedUranium.getLiquid(1000)).add("uranium_leaching",120,1000);
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java]: these are the same exact recipes

File: common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java [hunk 2]
```diff
@@ -233,4 +232,17 @@ private static void plasticPolymerization(Material monomer, Material polymer, in
...
+        CHEMICAL_REACTING.RB().fi(SodiumBicarbonateSolution.getLiquid(1000)).fo(SodiumCarbonateSolution.getLiquid(1000),CarbonDioxide.getGas(1000),AntimatterMaterials.Water.getLiquid(1000)).add("sodium_carbonate_solution",100,200);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Thorium,1))).fi(LeachingSolution.getLiquid(1000)).fo(LeachedThorium.getLiquid(1000)).add("thorium_leaching",120,1000);
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(Uranium,1))).fi(LeachingSolution.getLiquid(1000)).fo(LeachedUranium.getLiquid(1000)).add("uranium_leaching",120,1000);
+
+        CHEMICAL_REACTING.RB().ii(of(DUST.get(TrithoriumOctoxide,11))).fi(NitricAcid.getLiquid(3000)).fo(Thoriumdioxidedinitrate.getLiquid(3000),AntimatterMaterials.Water.getLiquid(4000),NitrogenDioxide.getGas(2000)).add("thoriumdioxidedinatrate",120,1000);
+        CHEMICAL_REACTING.RB().fi(Thoriumdioxidedinitrate.getLiquid(10000),Ammonia.getGas(22000)).fo(Ammoniumdithoranate.getLiquid(5000)).add("uranium_leaching",120,1000);
```
Comment (Trinsdar) [inline common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java]: ammonium dithoranate does not seem to have a gas or liquid

**Review by Trinsdar — CHANGES_REQUESTED** [review]

File: common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java (hunk 1 above)
Comment (Rundas01) [reply common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java]: true, must have happened while copy/pasting stuff. will remove the dupes ofc

File: common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java (hunk 2 above)
Comment (Rundas01) [reply common/src/main/java/muramasa/gregtech/loader/machines/ChemicalReactorLoader.java]: probably because they should be dusts

**Comment by Rundas01** [comment]
probably became unnecessary, due to the new PR I've opened. will close this once confirmed.

**Comment by Rundas01** [comment]
decame unecessary

**Pull request closed without merging**